manim/mobject/geometry.py

```diff
@@ -764,6 +764,33 @@ def generate_points(self):
 
 
 class Sector(AnnularSector):
+    """
+
+    Examples
+    --------
+
+    .. manim:: ExampleSector
+        :save_last_frame:
+
+        class ExampleSector(Scene):
+            def construct(self):
+                sector = Sector(outer_radius=2, inner_radius=1)
+                sector2 = Sector(outer_radius=2.5, inner_radius=0.8).move_to([-3, 0, 0])
+                sector.set_color(RED)
+                sector2.set_color(PINK)
```

Comment:
not necessary, as later `sector2.set_color(WHITE)` 
Also, please only use self.add once with all mobjects.
Further, self.wait() is not needed